Added all most possible   troubleshooting commands (#1150)
MicrosoftDocs/windows-powershell-docs: docset/windows/troubleshootingpack/invoke-troubleshootingpack.md

The second command invokes the troubleshooting pack stored in $Audio in unattended mode.
The command specifies an answer file, previously created by using the **Get-TroubleshootingPack** cmdlet.

> ### Troubleshooting commands  for all the components or devices and save all reports in C:\Diagresult

Reviewer: ```suggestion
### Troubleshooting commands for all the components or devices and save all reports in C:\Diagresult
```

> make a folder "DiagResult" in C:\

Reviewer: ```suggestion
Make a folder "DiagResult" in C:\
```

> then open powershell application with administrator right, type the following 

Reviewer: ```suggestion
Then open PowerShell with administrator permissions and use one of the following.
```

**Invoke-TroubleshootingPack -Pack $Search -Result "C:\DiagResult"**

15. Speech
**$Speech = Get-TroubleshootingPack -Path "C:\Windows\Diagnostics\System\Speech"**
**$SpeInvoke-TroubleshootingPack -Pack $Speech -Result "C:\DiagResult"**

16. Video
**$Video = Get-TroubleshootingPack -Path "C:\Windows\Diagnostics\System\Video"**
**Invoke-TroubleshootingPack -Pack $Video -Result "C:\DiagResult"**

17. WindowsmediaPlayerConfiguartion
**$WindowsMediaPlayerConfiguration = Get-TroubleshootingPack -Path "C:\Windows\Diagnostics\System\WindowsMediaPlayerConfiguration"**
**Invoke-TroubleshootingPack -Pack $WindowsMediaPlayerConfiguration -Result "C:\DiagResult"**

18. WindowsMediaPlayerMediaLibrary
**$WindowsMediaPlayerMediaLibrary = Get-TroubleshootingPack -Path "C:\Windows\Diagnostics\System\WindowsMediaPlayerMediaLibrary"**
**Invoke-TroubleshootingPack -Pack $WindowsMediaPlayerMediaLibrary -Result "C:\DiagResult"**

19. WindowsMediaPlayerPlayDVD
**$WindowsMediaPlayerPlayDVD = Get-TroubleshootingPack -Path "C:\Windows\Diagnostics\System\WindowsMediaPlayerPlayDVD"**
**Invoke-TroubleshootingPack -Pack $WindowsMediaPlayerPlayDVD -Result "C:\DiagResult"**

> **Note**:: All Troubleshooting reports will be  stored under this folder  **C:\DiagResult\**

Reviewer: ```suggestion
> [!NOTE]
>  All Troubleshooting reports will be  stored in the folder  **C:\DiagResult\**
```
